system/lib/libc/crt1.c

```diff
@@ -12,7 +12,7 @@
 #include <stdlib.h>
 #include <wasi/api.h>
 
-__attribute__((weak)) void __wasm_call_ctors(void);
+__attribute__((__weak__)) void __wasm_call_ctors(void);
```

Comment:
Is there a reason to prefer the underscores? I've never seen this form I don't think.

Reply:
Sorry I should have written more notes in the PR description.

This was needed because musl internally defines `weak` .. so for any code that gets compiled with musl itself we can't use the word `weak` anywhere.